frontend/src/components/NutritionFacts.jsx

```diff
@@ -0,0 +1,141 @@
+import ingredients from '../data/ingredients.json'
+import nutritionData from '../data/nutrition_data.json'
+import './NutritionFacts.css';
+
+export default function NutritionFacts(props) {
+
+    let calories = 0;
+
+    let sumNutrition = {
+        "Total Fat": {
+            "quantity": 0,
+            "unit": "g",
+            "dailyValue": 0,
+        },
+        "Cholesterol": {
+            "quantity": 0,
+            "unit": "mg",
+            "dailyValue": 0,
+        },
+        "Sodium": {
+            "quantity": 0,
+            "unit": "mg",
+            "dailyValue": 0,
+        },
+        "Carbohydrates": {
+            "quantity": 0,
+            "unit": "g",
+            "dailyValue": 0,
+        },
+        "Protein": {
+            "quantity": 0,
+            "unit": "g",
+            "dailyValue": 0,
+        },
+        "Vitamin-D": {
+            "quantity": 0,
+            "unit": "IU",
+            "dailyValue": 0,
+        },
+        "Calcium": {
+            "quantity": 0,
+            "unit": "mg",
+            "dailyValue": 0,
+        },
+        "Iron": {
+            "quantity": 0,
+            "unit": "mg",
+            "dailyValue": 0,
+        },
+        "Potassium": {
+            "quantity": 0,
+            "unit": "mg",
+            "dailyValue": 0,
+        },
+        "Vitamin-C": {
+            "quantity": 0,
+            "unit": "mg",
+            "dailyValue": 0,
+        },
+    }
+
+    function findIngredients(recipe) {
```

Comment:
Can you move this helper functions outside of the component (above or below)? It makes it hard to see what this function actually does when it has these definitions inside it. Since they don't depend on anything in the component, they should be outside so the component just has the definitions needed to render.